### ISSUE TITLE
IPE Accessing Latex Through the Cloud Broken

### ISSUE THREAD

**Issue by coveringspaces** [otfried/ipe#407]
All of a sudden, I get the following error when I try to compile IPE using Latex:

"Ipe did not succeed to run Latex to process your text objects.

You either have no Latex installation on your computer, or the Latex executable is not on your command path.

If you prefer, you can enable Latex-compilation in the cloud from the Help menu."

I had always been using Latex-compilation in the cloud and it always worked. I'm still connected to the internet and I even uninstalled and re-installed IPE. What might the issue be?

**Comment by otfried** [comment]
It works fine for me right now.  Perhaps the server was temporarily down.

Which platform are you on?  Do you have the most recent Ipe version (7.2.24)?   Did you check in the Help menu that online Latex conversion is indeed enabled?

**Comment by coveringspaces** [comment]
Hello Otfried, 

Thanks for the quick response. I'm on MacOS and I am pretty sure I have the most recent Ipe version as I uninstalled and reinstalled Ipe yesterday. (I also just checked the about menu and it says that I have Ipe 7.2.24.) 

As far as I can see, Latex conversion is enabled, as can be seen in the screenshot.

<img width="204" alt="Screen Shot 2021-11-01 at 12 21 47 PM" src="https://user-images.githubusercontent.com/26236365/139705227-16d56045-c5a0-4278-b0dc-3ec2789d2d74.png">

Here is the error message I am getting:

<img width="421" alt="Screen Shot 2021-11-01 at 12 19 43 PM" src="https://user-images.githubusercontent.com/26236365/139705273-1f6de07f-6549-4929-a886-2eea7d67e429.png">

Help in fixing this would be greatly appreciated!
